src/_guides/language/effective-dart/usage.md

```diff
@@ -1020,35 +1020,41 @@ The following best practices apply to asynchronous coding.
 
 ### PREFER async/await over using raw futures.
 
-Explicit asynchronous code is notoriously hard to read and debug, even when
-using a nice abstraction like futures. This is why we added `async`/`await` to
-the language. They make a huge improvement in readability code and let you use
-all of the built-in control flow structures of the language within your
-asynchronous code.
+Asynchronous code is notoriously hard to read and debug, even when using a nice
+abstraction like futures. This is why we added `async`/`await` to the language.
```

Comment:
Do we really want to bring language evolution into this discussion? cc @kwalrath WDYT?

How about more simply:

> Asynchronous code is notoriously hard to read and debug, even when using a nice abstraction like futures. The `async`/`await` syntax can contribute to improving code readability while letting you use all of the Dart control flow structures within your async code.

Reply:
Good point, adding async-await is ancient history.

Reply:
Good call. Simplified it even further.